Client/Frontend/Widgets/AutocompleteTextField.swift

```diff
@@ -39,9 +40,11 @@ class AutocompleteTextField: UITextField, UITextFieldDelegate {
 
     private func commonInit() {
         super.delegate = self
+        super.addTarget(self, action: "SELtextDidChange:", forControlEvents: UIControlEvents.EditingChanged)
         notifyTextChanged = debounce(0.1, {
             if self.editing {
-                self.autocompleteDelegate?.autocompleteTextField(self, didEnterText: self.text)
+                let enteredText = self.text.substringToIndex(advance(self.text.startIndex, self.enteredTextLength, self.text.endIndex))
+                self.autocompleteDelegate?.autocompleteTextField(self, didEnterText: enteredText)
```

Comment:
@thebnich  Here we need `enteredText` because it is also called from `SELtextDidChange`. Here we are having complete string with completion not the user entered text.




Updated the pull request.